ports/cpp/source/antlr4-c3/CodeCompletionCore.hpp

```diff
@@ -5,8 +5,8 @@
 //  Licensed under the MIT License.
 //
 
-#ifndef CodeCompletionCore_hpp
-#define CodeCompletionCore_hpp
+#ifndef CODE_COMPLETION_CORE_HPP
+#define CODE_COMPLETION_CORE_HPP
```

Comment:
Not an issue per se, but why not using `pragma once`, it's much nicer to read.

Reply:
Replaced with `pragma once`